apps/challenges/urls.py

```diff
@@ -181,4 +181,9 @@
         views.manage_worker,
         name="manage_worker",
     ),
+    url(
+        r"^get_all_challenge_templates$",
```

Comment:
Hey @KhalidRmb can you please add a `/` at the end of this url as we have all the api's ending with `/` only.

Reply:
Yes, will do.